ui/src/forms/matchers.ts

```diff
@@ -0,0 +1,19 @@
+import { cc } from '@cube-creator/core/namespace'
+import type { SingleEditor } from '@hydrofoil/shaperone-core'
+import { editor } from '@/forms/bulma'
+
+export const projectKindSelector: SingleEditor = {
+  term: editor.RadioButtons,
+  match (shape) {
+    const { in: choices, path } = shape
+    if (choices.length === 0) {
+      return 0
+    }
+    if (choices.length > 5) {
+      return null
+    }
```

Comment:
Do I understand this right that you can not have more than 5 options?

Reply:
If you look at DASH, it predefines a bunch of editors in its namespace. There is an enum selector which applies to `sh:in`. The spec wants it to render a dropdown. 

Here we give a higher score to enum of `<5` choices to prefer a radio button component.